st2common/bin/st2ctl

```diff
@@ -61,12 +61,20 @@ function validate_in_components() {
 
 function st2start() {
   for COM in ${COMPONENTS}; do
+    service_manager $COM status >/dev/null 2>&1;
+    if [[ "1" == $? ]]; then
```

Comment:
It's not exactly the same. `status` can give exit codes that are not 0 or 1; i.e. upstart gives some other code when you're trying to stop a service that's already stopped (or start a service that's already running). We probably don't want to hide those, so I only made it skip on 1 which is given when service is not found.